src/lib/updateGrantsGrantees.js

```diff
@@ -96,27 +98,45 @@ export async function processFiles() {
  *
  * Note - file download needs to happen in deployed environments
  */
-export default async function updateGrantsGrantees() {
-  try {
-    if (process.env.NODE_ENV === 'production') {
-      const response = await axios(process.env.HSES_DATA_FILE_URL, {
-        method: 'get',
-        url: process.env.HSES_DATA_FILE_URL,
-        responseType: 'stream',
-        auth: {
-          username: process.env.HSES_DATA_USERNAME,
-          password: process.env.HSES_DATA_PASSWORD,
-        },
+export default async function updateGrantsGrantees(_processFiles = processFiles) {
+  logger.info('updateGrantsGrantees: starting');
+  logger.debug('updateGrantsGrantees: retrieving file from HSES');
+  await axios(process.env.HSES_DATA_FILE_URL, {
+    method: 'get',
```

Comment:
Is `await` necessary here since you are using `.then`. My understanding was that you generally use one of the other since they do essentially the same thing. Is there a subtlety I am not aware of?

Reply:
Here, the `await` is waiting for the resolution of the Promise that is returned from the `then`.

We could probably rewrite it to be a series of `awaits` but this felt like a good way to bundle up the implementation details of piped streams in the `then` and all of our business logic in the top level `await`ed steps.